src/azul/service/manifest_controller.py

```diff
@@ -112,6 +112,9 @@ def get_manifest_async(self,
                     }
                     token = self.async_service.start_generation(state)
             else:
+                if fetch:
+                    raise BadRequestError('Query argument `objectKey` is reserved '
```

Comment:
The spec declares it for both fetch and non-fetch, AFAICT. Once we automatically validate incoming requests based on the spec, it would be weird to reject the request this late.